tests/src/test/scala/com/nvidia/spark/rapids/ParquetScaleTestSuite.scala

```diff
@@ -0,0 +1,548 @@
+    tab("c1").setNullProbability(0.8)
+    tab("c1").setLength(3)
+    tab("c1")("key").setLength(3)
+    tab("c1")("value").setLength(3)
+
+    def genDf(tab: TableGen): SparkSession => DataFrame = spark => tab.toDF(spark)
+
+    checkStats(genDf(tab))
+  }
+
+  test("Statistics tests for Parquet files written by GPU, map(struct)") {
+    assume(false, "Move to scale test")
+    val schema =
+      """
+      struct<
+        c1: map<
+          struct<
+            c101: long,
+            c102: int
+          >,
+          struct<
+            c101: long,
+            c102: string
+          >
+        >
+      >
+      """
+    val gen = DBGen()
+    gen.setDefaultValueRange(TimestampType, minTimestampForOrc, maxTimestampForOrc)
+    val tab = gen.addTable("tab", schema, rowsNum)
+    tab("c1").setNullProbability(0.8)
+    tab("c1").setLength(3)
+    tab("c1")("key").setLength(3)
+    tab("c1")("value").setLength(3)
+
+    def genDf(tab: TableGen): SparkSession => DataFrame = spark => tab.toDF(spark)
+
+    checkStats(genDf(tab))
+  }
+
+  test("Statistics tests for Parquet files written by GPU, array(map)") {
+    assume(false, "Move to scale test")
+    val schema =
+      """
+      struct<
+        c1: array<map<string, long>>
+      >
+      """
+    val gen = DBGen()
+    gen.setDefaultValueRange(TimestampType, minTimestampForOrc, maxTimestampForOrc)
+    val tab = gen.addTable("tab", schema, rowsNum)
+    tab("c1").setNullProbability(0.8)
+    tab("c1")("child").setLength(3)
+
+    def genDf(tab: TableGen): SparkSession => DataFrame = spark => tab.toDF(spark)
+    /**
+     * Note: There are discrepancies between CPU and GPU file schemas,
+     * but the Spark can read both them correctly, so it's not an issue.
+     *
+     * Details:
+     *
+     * CPU Parquet file schema is:
+     * message spark_schema {
+     *   optional group c1 (LIST) {
+     *     repeated group list {
+     *       optional group element (MAP) {
+     *         repeated group key_value {
+     *           required binary key (STRING);
+     *           optional int64 value;
+     *         }
+     *       }
+     *     }
+     *   }
+     * }
+     *
+     * GPU Parquet file schema is:
+     * message schema {
+     *   optional group c1 (LIST) {
+     *     repeated group list {
+     *       optional group c1 (MAP) {
+     *         repeated group key_value {
+     *           required binary key (STRING);
+     *           optional int64 value;
+     *         }
+     *       }
+     *     }
+     *   }
+     * }
+     *
+     * Spark reads both of them as:
+     *
+     * df.printSchema()
+     * root
+     *  |-- c1: array (nullable = true)
+     *  |    |-- element: map (containsNull = true)
+     *  |    |    |-- key: string
+     *  |    |    |-- value: long (valueContainsNull = true)
+     *
+     */
+    // skip check the schema
+    val (cpuStat, gpuStat) = checkStats(genDf(tab), skipCheckSchema = true)
```

Comment:
Instead of skipping the schema entirely, can we still check, but know that the top level message name does not match, and names of messages under lists will not match?  This can be a follow on issue if we want to.